buildspec_windows_create_manifest.yml

```diff
@@ -0,0 +1,53 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - echo "Creating manifest for aws-for-fluent-bit Windows image"
+      - yum update -y && yum upgrade -y
+      - yum install jq -y
+  build:
+    commands:
+      - |
+        export DOCKER_CLI_EXPERIMENTAL=enabled
+        declare -A os_releases=( ["windows2019"]="ltsc2019" ["windows2022"]="ltsc2022")
+        
+        # Push the image to ECR with corresponding architecture as the tag.
+        aws ecr get-login-password --region ${AWS_REGION}| docker login --username AWS --password-stdin ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
+        
+        # Find the applicable windows versions.
+        cd $CODEBUILD_SRC_DIR
+        versions=$(cat windows.versions | jq -c -r '.windows[].version')
+        latest_version=""
+        
+        # Create and push the image manifests.
+        while read -r version; do
+          if [[ "$latest_version" < "$version" ]]; then
+            latest_version=$version
+          fi
+          
+          docker manifest create ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:${version}-windowsservercore \
+          ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:${version}-${os_releases[windows2019]} \
+          ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:${version}-${os_releases[windows2022]}
+        
+          # Sanity check the manifest.
+          docker manifest inspect ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:${version}-windowsservercore
+          
+          # Push manifest to ECR.
+          docker manifest push ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:${version}-windowsservercore
+        done <<< "$(echo "$versions")"
+        
+        # Create and push manifest for latest image.
+        docker manifest create ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/amazon/aws-for-fluent-bit-windows:windowsservercore-latest \
```

Comment:
Where do the environment variables such as AWS_ACCOUNT, AWS_REGION get propagated from? The code pipeline upstream or just inherently in the code build project

Reply:
So these would be pumped in during the infra creation. 
When we would create the CodeBuild projects then these would be either set as environment variables.